program.h

```diff
@@ -12,12 +12,16 @@ struct program {
 	const char *label;
 	unsigned num_sectors;
 	unsigned partition;
+	bool sparse;
```

Comment:
```suggestion
	bool is_sparse;
```

nitpick: wonder if it makes sense to put these with `is_nand & is_erase` ?

Reply:
Not sure, "sparse" naming reflects a real XML-attribute "sparse", same as "label", "num_sectors" and etc